upgrade_tests/test_scenarios/test_client.py

```diff
@@ -42,11 +235,12 @@ class Scenario_preupgrade_client_package_installation(TestCase):
         4. Check if the package is installed on the pre-upgrade client
     """
     docker_vm = os.environ.get('DOCKER_VM')
-    ak = os.environ.get('RHEV_CLIENT_AK_RHEL7')
-    cv = entities.ActivationKey(organization=1).search(
-        query={'search': 'name={}'.format(ak)}
-    )[0].content_view
-    package_name = 'horse'
+    org = entities.Organization(id=1).read()
+    repo_url = 'http://inecas.fedorapeople.org/fakerepos/zoo3/'
```

Comment:
is this url one time used ? may be good to define at one place and then reuse further in other tests. but not a blocker to merge.